ide/csl.api/nbproject/org-netbeans-modules-csl-api.sig

```diff
@@ -807,14 +807,17 @@ hfds ADDED,MAX_KEEP,REMOVED,delta,editedEnd,edits,originalEnd,previous,start,tok
 hcls Edit
 
 CLSS public org.netbeans.modules.csl.api.EditList
+cons public init(javax.swing.text.Document)
```

Comment:
Result of @sdedic changes, I assume.

Reply:
OK, BaseDocument type deprecation in API.